nasa/harmony
# Harmony 1240 search after

## Jira Issue ID
HARMONY-1240

## Description
Replace use of the CMR scroll API with the search after API

## Local Test Steps
* Check out this branch and start Harmony
* Execute a query
* Verify the results are what you expect
* Look in your local staging bucket under the `SearchParams` folder and verify that you see a folder with a UUID for a name. Under that folder you should see a file named `serializedQuery` - this should have the JSON for your search request. This should work with shapefiles as well. The shapefile is already uploaded to S3, so you should just see a `geojson` field in the search JSON pointing to the url of the shapefile in S3.

## PR Acceptance Checklist
